src/main/resources/application.conf

```diff
@@ -23,7 +27,11 @@ swagger {
 }
 
 methods {
-  baseUrl="https://agora-ci.broadinstitute.org/methods"
+  baseUrl="https://agora-prod.broadinstitute.org/methods"
+}
+
+workspace {
+  workspacesUrl="https://rawls-dev.broadinstitute.org/workspaces"
```

Comment:
we might want to separate these out into base server and path before we get too far ... (a la https://github.com/broadinstitute/vault-api/blob/develop/application.conf)